resources/deletedb.yaml

```diff
@@ -1,3 +1,45 @@
+---
```

Comment:
`createdb`, `deletedb` and `hatest` is used only by developer. I do not see where we use it. Looks like Jeff and Anton have created it only on early development stage.